northstar/src/runtime/process/init.rs

```diff
@@ -40,11 +36,130 @@ pub(super) struct Init {
     pub seccomp: Option<AllowList>,
 }
 
+/// This is the type used to communicate with the init process from the runtime.
+#[derive(Debug, Serialize, Deserialize)]
+pub enum InitMessage {
+    Forked { pid: i32 },
+    Exit { pid: i32, exit_status: ExitStatus },
+}
+
+/// Init commands
+#[derive(Debug, Serialize, Deserialize)]
+pub enum InitApi {
```

Comment:
why is InitApi an enum?

Reply:
I thought there could be other potential commands for the init in addition to `Exec`. One option would be the `Shutdown` command. But at the moment it is not 100% clear for me how the shutdown  should behave.

Reply:
I think there's no need for a 'Shutdown'. The `init`lives while any child is alive. With the exit of the last child the init exits. A "shutdown" is a signal to the process group. The `init` itself doesn't handle signals and the childs exit. With the last child init exits as well...